docs/File-Formats.md

```diff
@@ -1375,3 +1376,88 @@ Example with 2 generic entities and 3 samples:
 <tr><td>mut_sig_mean</td><td>name of mean</td><td>description of mean</td><td>0.370</td><td>0.010</td><td>0.005</td></tr>
 <tr><td>mut_sig_confidence</td><td>name of confidence</td><td>description of confidence</td><td>0.653</td><td>0.066</td><td>0.050</td></tr>
 </table>
+
+## Resource Data
+
+The resource data is used to capture resouce data in patients, samples and studies.  
+
+the resource file is split into a resource difinition file, sample resource file, patient resource file and study resource file. All data files are required to have a matching meta file.
```

Comment:
good to run a spellchecker e.g. `difinition` -> `definition`

Reply:
a command line checker is `aspell` https://www.howtoforge.com/linux-aspell-command/

Reply:
we could maybe add that one as part of CI at some point ☝️

Reply:
This is pretty cool, good to know this!